### ISSUE TITLE
TypeError: 'module' object is not callable

### ISSUE THREAD

**Issue by cogphn** [freakboy3742/pyxero#226]
In my private application:

`
import datetime
from xero import Xero
from xero.auth import PrivateCredentials

with open("__path__") as keyfile:
    rsa_key = keyfile.read()
credentials = PrivateCredentials("__key__", rsa_key)
xero = Xero(credentials)
contacts = xero.contacts.filter(since=datetime(2013, 1, 1))
`

gives the following error:

TypeError: 'module' object is not callable. However it has no problem with this:

`
contacts = xero.contacts.all()
`

is this an issue or am I doing something wrong?

**Comment by g-yorke** [comment]
Had the same issue, I think the example on the project page is missing a datetime since it's both the module name and a class name -- this should work:

contacts = xero.contacts.filter(since=datetime.datetime(2013, 1, 1))

**Comment by Wozzyy** [comment]
Correct. `datetime` is a module which contains a class called `datetime` so you either need to do:

```python3
import datetime
x = datetime.datetime(2015, 10, 21)
```

or you _can_ do (but really shouldn't):

```python3
from datetime import datetime
x = datetime(2015, 10, 21)
```

The second one works and _might_ make your code less verbose but means you lose access to the rest of the datetime library like `datetime.timedelta` (perhaps you want Xero objects modified in the last 10 days so you could use `datetime.datetime.now() - datetime.timedelta(days=10)`


---

Either way, not a bug in PyXero, though I might send through a PR to amend the README to use `datetime.datetime`

**Comment by freakboy3742** [comment]
Closing as the question has been resolved.
